lib/lzo1x.js

```diff
@@ -0,0 +1,611 @@
+/*
+ * minilzo-js
+ * JavaScript port of minilzo by Alistair Braidwood
+ *
+ *
+ * This library is free software; you can redistribute it and/or
+ *  modify it under the terms of the GNU General Public License as
+ *  published by the Free Software Foundation; either version 2 of
+ *  the License, or (at your option) any later version.
+ *
+ * You should have received a copy of the GNU General Public License
+ *  along with the minilzo-js library; see the file COPYING.
+ *  If not, write to the Free Software Foundation, Inc.,
+ *  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ */
+
+/*
+ * original minilzo.c by:
+ *
+ * Markus F.X.J. Oberhumer
+ * <markus@oberhumer.com>
+ * http://www.oberhumer.com/opensource/lzo/
+ */
+
+/*
+ * NOTE:
+ *   the full LZO package can be found at
+ *   http://www.oberhumer.com/opensource/lzo/
+ */
+// This code is from https://github.com/abraidwood/minilzo-js. If it changes (unlikely, since it
```

Comment:
Have you opened an issue on the library's GitHub to see if we can get it fixed so that we don't have to use our own custom build?